python/hsfs/feature_view.py

```diff
@@ -1033,7 +1055,7 @@ def get_batch_data(
             start_time,
             end_time,
             self._batch_scoring_server.training_dataset_version,
-            self._batch_scoring_server._transformation_functions,
+            self._batch_scoring_server._model_dependent_transformation_functions,
```

Comment:
need to include ODT as well?

Reply:
While getting batch data the on-demand featuers are not re-computed but read directly from the feature group. So it is not actually required to pass ODT's.